tests/adapter_tests_async/test_async_falcon.py

```diff
@@ -0,0 +1,212 @@
+import json
+from time import time
+from urllib.parse import quote
+
+import falcon
+from falcon import testing
+
+from slack_sdk.signature import SignatureVerifier
+from slack_sdk.web.async_client import AsyncWebClient
+
+from slack_bolt.adapter.falcon.async_resource import AsyncSlackAppResource
+from slack_bolt.app.async_app import AsyncApp
+from slack_bolt.oauth.async_oauth_settings import AsyncOAuthSettings
+from tests.mock_web_api_server import (
+    setup_mock_web_api_server,
+    cleanup_mock_web_api_server,
+    assert_auth_test_count,
+)
+from tests.utils import remove_os_env_temporarily, restore_os_env
+
+
+def new_falcon_app():
+    if falcon.version.__version__.startswith("2."):
```

Comment:
You can remove this if condition

Reply:
Done in https://github.com/slackapi/bolt-python/pull/614/commits/515345c42aa1ca12034650f5d231829d4d27adf5